localization/interest_point/include/interest_point/BAD.h

```diff
@@ -0,0 +1,99 @@
+/**
+ * @copyright 2021 Xoan Iago Suarez Canosa. All rights reserved.
+ * Constact: iago.suarez@thegraffter.com
+ * Software developed in the PhD: Low-level vision for resource-limited devices
```

Comment:
what license?

Reply:
and does this really need to be included instead of linked to

Reply:
Done